main.go

```diff
@@ -153,11 +125,11 @@ func initialize() error {
 	}
 
 	d2term.BindLogger()
-	d2term.BindAction("dumpheap", "dumps the heap to heap.out", func() {
-		fileOut, _ := os.Create("heap.out")
+	d2term.BindAction("dumpheap", "dumps the heap to pprof/heap.pprof", func() {
+		os.Mkdir("./pprof/", 0755)
+		fileOut, _ := os.Create("./pprof/heap.pprof")
```

Comment:
I'd love to get an auto increment on the file name so is easy to do multiple dumps.

Reply:
Agreed, can you do this now or do you want me to go ahead and merge @Intyre ?